config/application.rb

```diff
@@ -6,14 +6,14 @@
 # Pick the frameworks you want:
 require 'active_model/railtie'
 require 'active_job/railtie'
-# require 'active_record/railtie'
-# require 'active_storage/engine'
+# require "active_record/railtie"
+# require "active_storage/engine"
 require 'action_controller/railtie'
-# require 'action_mailer/railtie'
-# require 'action_mailbox/engine'
-# require 'action_text/engine'
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
 require 'action_view/railtie'
-# require 'action_cable/engine'
+# require "action_cable/engine"
 require 'rails/test_unit/railtie'
 
 # Require the gems listed in Gemfile, including any gems
```

Comment:
On line 26, you'll prolly want to load the `8.0` defaults

Reply:
right, thanks ... forgot to change this